test/sql/CMakeLists.txt

```diff
@@ -98,6 +98,9 @@ if (CMAKE_BUILD_TYPE MATCHES Debug)
     timestamp_limits.sql
     with_clause_parser.sql
   )
+  list(APPEND TEST_TEMPLATES
```

Comment:
Is that change of this PR?

Reply:
just  combines the two if (debug) into 1